### PR TITLE
spring-fix:是否通知执行人

### PR DESCRIPTION
**Pull request by hardymgchen**
spring-fix:是否通知执行人

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=3984) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-sops?pullRequest=3984) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-sops?pullRequest=3984) it.</sub>

File: pipeline_plugins/tests/components/collections/sites/open/bk_test/notify/test_v1_0.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(self, cc_search_business_return=None, cmsi_send_msg_return=None):
 NOTIFY_HANDLE_API_ERROR = "gcloud.utils.cmdb.handle_api_error"
 BK_HANDLE_API_ERROR = "pipeline_plugins.components.collections.sites.open.bk.notify.v1_0.bk_handle_api_error"
 
-COMMON_PARENT = {"executor": "admin", "biz_cc_id": 2, "biz_supplier_account": 0}
+COMMON_PARENT = {"executor": "test", "biz_cc_id": 2, "biz_supplier_account": 0}
```
Comment (normal-wls) [inline pipeline_plugins/tests/components/collections/sites/open/bk_test/notify/test_v1_0.py:54]: 这里改成tester

File: pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py [hunk 2]
```diff
@@ -98,12 +103,19 @@ def execute(self, data, parent_data):
         if not result["result"]:
             data.set_outputs("ex_data", result["message"])
             return False
+        if notify == "notify_is_true":
```
Comment (normal-wls) [inline pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py:106]: 这里的写法优化下，直接用a if notify else b即可

File: pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py [hunk 3]
```diff
@@ -67,7 +67,11 @@ def inputs_format(self):
             self.InputItem(name=_("通知内容"),
                            key="bk_notify_content",
                            type="string",
-                           schema=StringItemSchema(description=_("通知的内容")))]
+                           schema=StringItemSchema(description=_("通知的内容"))),
+            self.InputItem(name=_("通知执行人"),
+                           key="notify",
+                           type="string",
```
Comment (normal-wls) [inline pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py:73]: 这里改成布尔

File: pipeline_plugins/components/static/components/atoms/bk/notify/v1_0.js [hunk 4]
```diff
@@ -151,6 +151,24 @@
                 ]
             }
         },
+        {
+            tag_code: "notify",
+            type: "radio",
+            attrs: {
+                name: gettext("通知执行人"),
+                items: [
+                    {value: "notify_is_true", name: gettext("是")},
```
Comment (normal-wls) [inline pipeline_plugins/components/static/components/atoms/bk/notify/v1_0.js:160]: 这里value直接改成true和false

File: pipeline_plugins/components/static/components/atoms/bk/notify/v1_0.js [hunk 5]
```diff
@@ -151,6 +151,24 @@
                 ]
```
Comment (normal-wls) [inline pipeline_plugins/components/static/components/atoms/bk/notify/v1_0.js:151]: static目录下的静态文件已提交一下同步修改

**Review by normal-wls — CHANGES_REQUESTED** [review]

**Review by normal-wls — APPROVED** [review]

**Pull request merged**